.github/workflows/bf-review.yml

```
@@ -29,8 +29,8 @@ env:
   BASE_DOMAIN: ${{ secrets.BASE_DOMAIN_STAGING }}
   DATABASE_USER: user
   DATABASE_PASSWORD: testing-password
-  K8S_REQUEST_CPU: 50m
-  K8S_REQUEST_RAM: 256Mi
+  K8S_REQUEST_CPU: 5m
+  K8S_REQUEST_RAM: 32Mi
```

```suggestion
  K8S_REQUEST_RAM: 256Mi
```

No need to reduce, apps use currently 150-250 Mi of memory